library/Rules/NoWhitespace.php

```diff
@@ -5,6 +5,10 @@ class NoWhitespace extends AbstractRule
 {
     public function validate($input)
     {
+        if (is_array($input)) {
+            return false;
+        }
+
         return is_null($input) || !preg_match('#\s#', $input);
```

Comment:
Actually what I did mean was to just replace the `preg_match(..` for `(is_scalar($input) && preg_match(..`.